src/index.html

```diff
@@ -11,7 +11,190 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <fieldset class="fieldset">
+
+        <legend>Personal information</legend>
+
+        <div class="div">
```

Comment:
Why use this class if it is not defined in CSS ? :)

Reply:
Oops :D